CONTRIBUTING.md

```diff
@@ -318,40 +345,37 @@ You will also need several other mkdocs plugins, like `branchcustomization` and
 To install all of this make sure you have a recent version of Python on your computer.
 The [DataLad Handbook](http://handbook.datalad.org/en/latest/intro/installation.html#python-3-all-operating-systems) provides helpful instructions for setting up Python.
 
-An easy way to install the correct version of mkdocs and all the other required extensions
-is to use the `requirements.txt` file contained in this repository,
-by using the following command:
+In general, we strongly recommend that you install all dependencies in an isolated Python environment.
+For example using `conda`, as described in the section on Python in the DataLad Handbook, linked above.
```

Comment:
```suggestion
For example using `conda`, as described in the [DataLad Handbook](http://handbook.datalad.org/en/latest/intro/installation.html#conda).
```
However, I'm not a fan of this handbook because it is biased towards using DataLad.

Reply:
If you have an application agnostic introduction to Python installation that you'd rather link to here, we're open to suggestions! The DataLad Handbook just finds a nice balance between verbose (good for newcomers) and still to the point, IMHO.

Reply:
I thought an explicit link would be better. But after browsing it, I didn't find it clear. Maybe [something like this one](https://geohackweek.github.io/Introductory/01-conda-tutorial/) would be better. But I am not a conda user.

Reply:
Thanks for the suggestion, I like the geohackweek resource for a quick `conda` intro -- it seems concise and a bit more helpful than the DataLad Handbook resource for this case. I'll add it.